server/apps/io/feeding_services/reuters.py

```diff
@@ -61,67 +46,38 @@ def _update(self, provider):
         if not last_updated or last_updated < updated - datetime.timedelta(minutes=ttl_minutes):
             last_updated = updated - datetime.timedelta(minutes=ttl_minutes)
 
-        for channel in self.get_channels():
-            for guid in self.get_ids(channel, last_updated, updated):
+        self.provider = provider
+        self.URL = provider.get('config', {}).get('url')
```

Comment:
problem is those urls could be hard to find, it's part of developers documentation, not sure what exactly you get when you subscribe. so keeping it could make things simpler, maybe as default values user could change via config?